### DISCUSSION TITLE
Ton 挂机升级小游戏

### DISCUSSION THREAD

**Discussion by workbzw** [NonceGeek/indiehacker-handbook#18] [IdeaStack]
挂机打怪升级小游戏

**Comment by leeduckgo** [comment]
> 挂机打怪升级小游戏

很好，相对于 Thread 那个，这个更适合起步。

**Comment by workbzw** [comment]
![MyHeroes](https://github.com/user-attachments/assets/0e6bdfaa-04ec-4991-ab0f-e74d922beb9f)

**Reply by leeduckgo** [reply]
> ![MyHeroes](https://private-user-images.githubusercontent.com/83118992/373319043-0e6bdfaa-04ec-4991-ab0f-e74d922beb9f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwMDc1MDgsIm5iZiI6MTcyODAwNzIwOCwicGF0aCI6Ii84MzExODk5Mi8zNzMzMTkwNDMtMGU2YmRmYWEtMDRlYy00OTkxLWFiMGYtZTc0ZDkyMmJlYjlmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDAyMDAwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4NzVkMjFlNjc3MDQ3MTNhMjY3M2VhMzg0M2E4MzcwZDQ3ZDhkOTM2NjNmNjRkYzI4ZjRjOGNiNzZkOTU5YmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.e1OSp50CA4Ugkzrp_x05Vso0xoEuhGKMchOb0FK2GUM)

先从文字版本的开始做起，这样不用操心美术素材的事情。

**Comment by workbzw** [comment]
repo:https://github.com/workbzw/my-heroes

**Reply by leeduckgo** [reply]
> repo:https://github.com/workbzw/my-heroes

Good beginning! I think the NFT renderer could refer the "loot"
